gast_to_code/gast_to_code_main.py

```diff
@@ -168,15 +167,15 @@ def js_unaryOp(gast):
 javascript: js
 python: py
 """
-def gast_router(gast, out_lang):
+def gast_to_node_router(gast, out_lang):
```

Comment:
Could this be `gast_to_code` or `convert_gast_to_code`?

Reply:
+1. Router is an implementation detail which the caller should not care about.

Reply:
Updated!